### ISSUE TITLE
Should I still use this?

### ISSUE THREAD

**Issue by msageryd** [ide/react-native-button#76]
Hi James and other maintainers.
This button is awesome and serves me well in my projects. But the RN-world keeps spinning and there is now a Button component in RN core.

I prefer this button over the built in button. But I wonder if I should switch to a more modern button? I'm not using Expo (yet). @ide, What button component do you use in Expo?

**Comment by ide** [comment]
Gesture Handler (https://github.com/kmagiera/react-native-gesture-handle) is probably the best button primitive. The component code of a button is pretty basic so I don't think it matters much what you use compared to the foundational pieces. Gesture Handler provides a good foundation for touchable views including buttons and IIRC has some button components or components you can use to make buttons.

**Comment by msageryd** [comment]
Thanks, I'll check it out. 
Corrected link: https://github.com/kmagiera/react-native-gesture-handler

**Comment by msageryd** [comment]
@ide Sorry for using this thread for OT, but I couldn't find any issue-registration on expo.io.

Tried to run @osdnk/gesturehandlerexample in the Expo client for ios. Got the email link, but Expo tells me that I can only run my own projects.

I tried to run the the app in Appetize instead, but only got to the web page for the project. If this is not the intended behaviour of the Expo client, you might want to look into it. If you think this is an issue with the gesture-handler example project, I'll post it there.
